packages/manager/apps/public-cloud/src/index.js

```diff
@@ -0,0 +1,28 @@
+import 'script-loader!jquery'; // eslint-disable-line
+import 'script-loader!lodash'; // eslint-disable-line
+
+import angular from 'angular';
+import core from '@ovh-ux/manager-core';
```

Comment:
Prefer named as:
```suggestion
import ovhManagerCore from '@ovh-ux/manager-core';
```

In order to make consistency with the README file.